cpp/tests/spatial/pip_test.cu

```diff
@@ -101,6 +101,24 @@ TYPED_TEST(PIPTest, piptest)
     gettimeofday(&t2, nullptr);
     float gpu_pip_time2=cuspatial::calc_time("GPU PIP time 2(including point data transfer and kernel time)......",t1,t2);
 
+    int err_cnt=0,non_zero=0;
+    for(int i=0;i<this->point_len;i++)
```

Comment:
Should be using google mock tools for comparisons. See the changes I made to pip test last week.

Reply:
I could not locate your code. I vaguely remember gtest does have something specifically for it but  can not recall the details.